eq-author-api/utils/jwtVerifier.js

```diff
@@ -0,0 +1,32 @@
+const admin = require("firebase-admin");
```

Comment:
Can we move this to next to the middleware as it is not a generic utility

Reply:
Moved